src/components/alert/alert.stories.js

```diff
@@ -126,6 +126,4 @@ storiesOf('Alert', module)
     notes: { markdown: notes }
   })
   .add(...makeStory('default', dlsThemeSelector))
-  .add(...makeStory('classic', classicThemeSelector, true))
-  .add(...makeButtonStory('with button', dlsThemeSelector))
```

Comment:
I don't think we need to remove this line also, it is for opening `Alert` default